docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -1806,6 +1806,38 @@ Total number of bytes available to this Java virtual machine on all file
 stores. Depending on OS or process level restrictions (e.g. XFS quotas), this might appear
 less than `free_in_bytes`. This is the actual amount of free disk
 space the {es} node can utilise.
+
+`low_watermark_free_space`::
+(<<byte-units,byte value>>)
+Total amount of effective low watermark free disk space available to this Java virtual machine on this file store.
```

Comment:
Wording suggestion

```suggestion
The effective low disk watermark for this data path on this node: when a node
has less free space than this value for at least one data path, its disk usage
has exceeded the low watermark. See <<disk-based-shard-allocation>> for more
information about disk watermarks and their effects on shard allocation.
```

Reply:
OK, I will update other threshold types based on the suggestion, and also add them to `data` subsection.